conf/app.routes

```diff
@@ -8,6 +8,7 @@ POST    /upscan/upload                              controllers.upscan.UploadFor
 POST    /validate-submission                        controllers.validation.SubmissionValidationController.validateSubmission
 
 POST    /submit                                     controllers.SubmissionController.submitDisclosure
+POST    /submitxml                                  controllers.SubmissionController.submitDisclosureXML
```

Comment:
Can you move this to a testOnlyRoute as well please, and update app-config-staging to use testOnly routes for the backend. The frontend already does.